[Docs] Minor fix to apply_ac doc

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #85459
* #85449
* __->__ #85448

Per title

Created from CodeHub with https://fburl.com/edit-in-codehub

Differential Revision: [D39714530](https://our.internmc.facebook.com/intern/diff/D39714530/)

rohan-varma committed:
[Docs] Minor fix to apply_ac doc

Per title

Created from CodeHub with https://fburl.com/edit-in-codehub

Differential Revision: [D39714530](https://our.internmc.facebook.com/intern/diff/D39714530/)

[ghstack-poisoned]

awgu commented on torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py:
```diff
@@ -227,13 +227,13 @@ def apply_activation_checkpointing_wrapper(
         check_fn (Optional[Callable[nn.Module, nn.Module]])
-            A lambda function which will be passed current layer and returns
-            ``True`` or ``False`` depending on whether input layer should be wrapped.
+            A lambda function which will be passed each child of `model` and returns
```
nit: 
```suggestion
            A lambda function which will be passed each child submodule of ``model`` and returns
```

awgu commented on torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py:
```diff
@@ -227,13 +227,13 @@ def apply_activation_checkpointing_wrapper(
+        model (nn.Module):
+            The model whose submodules should be wrapped with activation checkpointing.
         checkpoint_wrapper_fn (Optional[Callable[nn.Module]])
             A `Callable` which will wrap modules
```
nit (before PR): 
```suggestion
            A ``Callable`` which will wrap modules
```
I think single backtick in docstrings are rendered as italicized, while double backtick is code type.

awgu reviewed (approved):
LGTM, but consider the nits before landing :)

rohan-varma committed:
Update on "[Docs] Minor fix to apply_ac doc"


Per title

Created from CodeHub with https://fburl.com/edit-in-codehub

Differential Revision: [D39714530](https://our.internmc.facebook.com/intern/diff/D39714530/)

[ghstack-poisoned]
